syne_tune/optimizer/schedulers/multiobjective/multi_surrogate_multi_objective_searcher.py

```diff
@@ -119,16 +120,30 @@ def __init__(
             # overwritten in ``_restore_from_state``
             self._create_internal(**kwargs.copy())
 
+    def _set_metric_to_internal_signs(self):
```

Comment:
Why do we have to deal with this here? Should this be done in the BO module already?